go.mod

```diff
@@ -2,4 +2,16 @@ module github.com/ipfs/fs-repo-migrations
 
 go 1.13
 
-require golang.org/x/net v0.0.0-20200114155413-6afb5195e5aa
+require (
+	github.com/ipfs/go-blockservice v0.1.3
+	github.com/ipfs/go-datastore v0.4.5
+	github.com/ipfs/go-ds-flatfs v0.4.5
+	github.com/ipfs/go-ds-leveldb v0.4.2
+	github.com/ipfs/go-filestore v1.0.0
+	github.com/ipfs/go-ipfs-blockstore v1.0.1
+	github.com/ipfs/go-ipfs-exchange-offline v0.0.1
+	github.com/ipfs/go-ipfs-pinner v0.0.5-0.20201029233447-cad837873ee6
+	github.com/ipfs/go-ipld-format v0.2.0
+	github.com/ipfs/go-merkledag v0.3.2
+	golang.org/x/net v0.0.0-20200114155413-6afb5195e5aa
+)
```

Comment:
@Stebalien WDYT is this fine for now and then we move to implementing https://github.com/ipfs/fs-repo-migrations/issues/98, or should we just do that now?

Reply:
Note: there might be alternatives to #98 such as vendoring the dependencies and giving them all migration-specific names so we can keep everything in one binary, but I'm not sure if they'd be any better.

Reply:
We can't have unvendored deps in this repo but we could try using go mod vendoring.

Reply:
> The go mod vendor command constructs a directory named vendor in the main module's root directory that contains copies of all packages needed to support builds and tests of packages in the main module.

Is the suggestion to make new packages + go.mod's in each migration and then to try and vendor, or to copy the approach from the migrations before we were using gx.